### ISSUE TITLE
Tabs are 2 tab stops on first interaction, then it functions normally afterward

### ISSUE THREAD

**Issue by andrew-poblete** [Workday/canvas-kit#2152]
## 🐛 Bug Report

When navigating tabs, the first and second tabs now receive focus, which is not how tabs should function. After the 2nd tab receives focus, the tab set reverts to acting like standard tabs.

## To Reproduce

1. https://canvas.workday.com/v8/assets/system-icons (v8 site)
2. Inspect the Guidance/Gallery/Create Your Own tabs (notice that they all have tabindex=0 set)
3. Tab to Guidance.
4. Tab one more time. Focus moves to Gallery and the tabindex in the code becomes -1.

## Expected Behavior
The tablist should be one tab stop and the tabs should only be accessible using the arrow keys.

## Actual Results
Focus moves to the first tab, then pressing tab again moves focus to the second tab instead of out of the tablist.

## Browser (if applicable)

Chrome

## Link to repl or repo (highly encouraged)

N/A

### Error Output

N/A

Include a screenshot of a visual bug
![Capture](https://user-images.githubusercontent.com/105951517/233490970-a352968d-0e6a-4f6f-87bc-c7f27e23ef03.png)
